plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useRef, useState } from 'react'
+import { observer } from 'mobx-react'
+import { makeStyles } from 'tss-react/mui'
+import { colord } from '@jbrowse/core/util/colord'
+import { ParsedLocString, getSession } from '@jbrowse/core/util'
+import { Menu } from '@jbrowse/core/ui'
+import { Base1DViewModel } from '@jbrowse/core/util/Base1DViewModel'
+import { IconButton, Tooltip } from '@mui/material'
+
+// icons
+import LinkIcon from '@mui/icons-material/Link'
+
+// locals
+import { LinearGenomeViewModel } from '../model'
+
+type LGV = LinearGenomeViewModel
+
+const COLOR = 'rgb(218, 165, 32)'
+
+const useStyles = makeStyles()({
+  highlight: {
+    height: '100%',
+    position: 'absolute',
+    textAlign: 'center',
+    overflow: 'hidden',
+    display: 'flex',
+    alignItems: 'start',
+  },
+})
+
+interface ParsedLocStringA {
+  assemblyName: string
+  refName: string
+  start: number
+  end: number
+  reversed: boolean
+}
+
+const Highlight = observer(function Highlight({ model }: { model: LGV }) {
+  const { classes } = useStyles()
+  const [open, setOpen] = useState(false)
+  const anchorEl = useRef(null)
+
+  const session = getSession(model)
+
+  const menuItems = [
+    {
+      label: 'Dismiss highlight',
+      onClick: () => model.setHighlight({} as ParsedLocString),
+    },
+  ]
```

Comment:
@cmdcolin 
Made the icon/onhover element of the highlight into a icon button -> opens ability to dismiss the highlight.

We could consider tossing this:

```
    {
      label: 'Bookmark highlighted region',
      icon: BookmarkIcon,
      onClick: () => {
        bookmarkWidget.addBookmark(model.highlight as Region)
      },
    },
```

kind of thing in there too, but I anticipate with the current setup that would necessitate: https://github.com/GMOD/jbrowse-components/pull/4003/files#r1406162147

Thoughts on whether that would be a good addition to this PR?

Reply:
that would be fine, it might have to detect if bookmarkwidget is available